doc/manual.rst

```diff
@@ -313,6 +313,38 @@ it was not case-sensitive and underscores were ignored and there was not even a
 distinction between ``foo`` and ``Foo``.
 
 
+Stropping
+---------
+
+`Stropping <https://en.wikipedia.org/wiki/Stropping_(syntax)>`_
+allows the same letter sequence to be used both as a keyword and as an identifier, this simplifies parsing and
+`FFI <https://en.wikipedia.org/wiki/Foreign_function_interface>`_
+with languages where that identifier is not a reserved keyword.
```

Comment:
Total alien style, the Nim manual doesn't introduce general concepts and then specializes it for how it's done in Nim. Also, I would never use the word "stropping" as I have no idea what it means. This should be written like this:

Keywords as identifiers
-------------------------

If a keyword is enclosed in backticks it loses its keyword property and becomes an ordinary identifier.


^ Clear, to the point, doesn't digress into Wikipedia territory.